openassessment/templates/base.html

```diff
@@ -0,0 +1,3 @@
+<div>
+  <h1>Hello world</h1>
+</div>
```

Comment:
What is the purpose of this file?

Reply:
That is going to have an iframe to the new xblock UI